ui/admin/app/components/form/auth-method/oidc/index.hbs

```diff
@@ -406,45 +397,45 @@
 
   </Form::Field::ListWrapper>
 
-  <form.input
-    @name='max_age'
-    @type='number'
+  <Hds::Form::TextInput::Field
     @value={{@model.max_age}}
-    @label={{t 'form.max_age.label'}}
-    @helperText={{t 'form.max_age.help'}}
-    @error={{@model.errors.max_age}}
-    readonly={{false}}
+    @isInvalid={{@model.errors.max_age}}
     step='1'
     min='0'
```

Comment:
Step and min are only valid for number input types

Reply:
@ZedLi good catch incoming change

Reply:
I don't think the change you made changed anything? It's still setting them

Reply:
@ZedLi fixed in last commit